### PR TITLE
[pimcore-11] BatchOperationBundle v2.0.0

### PR DESCRIPTION
**Pull request by Alanin**
update bundle to pimcore 11

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,14 +24,15 @@
       ]
     },
     "patches": {
-      "pimcore/pimcore": {
+      "pimcore/admin-ui-classic-bundle": {
         "Add buttons to grid view": "vendor/studio1/batch-operation/patches/listfolder.patch"
```
Comment (isekep) [inline composer.json:28]: @Alanin Is this still necessary?! The latest version v1.1.2 of pimcore/admin-ui-classic-bundle introduced a huge refactoring... thx

File: src/BatchOperationBundle.php [hunk 2]
```diff
@@ -13,40 +13,54 @@
...
+        }
     }
 
     /**
      * @inheritDoc
      */
-    public function getDescription()
+    public function getDescription(): string
     {
         return 'Add addtional asset batch operations for tagging and moving files';
```
Comment (isekep) [inline src/BatchOperationBundle.php:57]: @Alanin  Please fix typo `addtional` -> `additional`, update `composer.json` and repo description as well. Thx

**Review by isekep — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,10 +2,13 @@
...
 
-| **Bundle Version**  | **Pimcore Version** |
-|---------------------|---------------------|
-| 1.x.x               | Pimcore 10          |
+| **Bundle Version** | **Pimcore Version** |
+|--------------------|---------------------|
+| 1.x.x              | Pimcore 10          |
+| 2.x.x              | Pimcore 11          |
 
+## [2.0.0] - 2023-09-27
```
Comment (isekep) [inline CHANGELOG.md:10]: @Alanin Please update pr title/description, thx

**Review by isekep — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (Alanin) [reply composer.json:28]: yep, buttons were not added to the GUI without that patch

File: src/BatchOperationBundle.php
Comment (isekep) [inline src/BatchOperationBundle.php:1]: 👍  Cool idea to use `PackageVersionTrait`, @Alanin

Comment (isekep) [reply src/BatchOperationBundle.php:1]: ..maybe we should use this more often / in other Studio1® pimcore bundles

**Review by isekep — APPROVED** [review]

**Pull request merged**